### PR TITLE
added sensor indicating remote instance conection state

### PR DESCRIPTION
**Pull request by jacekpaszkowski**
regarding #35

### CODE REVIEW

File: custom_components/remote_homeassistant/__init__.py [hunk 1]
```diff
@@ -210,6 +234,8 @@ def _next_id(self):
             self._hass.states.async_remove(entity)
         if self._remove_listener is not None:
             self._remove_listener()
+        if self._connection_state_entity is not None:
```
Comment (lukas-hetzenecker) [inline custom_components/remote_homeassistant/__init__.py]: probably not needed?

File: custom_components/remote_homeassistant/__init__.py [hunk 2]
```diff
@@ -47,6 +54,7 @@
 
 INSTANCES_SCHEMA = vol.Schema({
     vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_STATE_ENTITY_NAME): cv.string,
```
Comment (lukas-hetzenecker) [inline custom_components/remote_homeassistant/__init__.py]: please don't add a new required attribute, as this adds a breaking change
The name of the sensor could be automatcally generated, like `sensor.remote_connection_<host>_<port>`

File: custom_components/remote_homeassistant/__init__.py (hunk 1 above)
Comment (jacekpaszkowski) [reply custom_components/remote_homeassistant/__init__.py]: fixed

File: custom_components/remote_homeassistant/__init__.py (hunk 2 above)
Comment (jacekpaszkowski) [reply custom_components/remote_homeassistant/__init__.py]: changed, please see new commit :)

**Comment by rchamp26** [comment]
hi there. i've been testing this for the past month or so and am seeing the same issue. 

i've tried 2.1 as well as the master branch. after about an hour, i no longer receive any state change updates from the Remote HA system (though if i look at the remote HA system, it operates as expected)

on my main HA instance, i see the following error indicating that the websocket is closed and seems to never recover until i restart HA

```Logger: custom_components.remote_homeassistant
Source: custom_components/remote_homeassistant/__init__.py:259
Integration: Remote Home-Assistant (documentation)
First occurred: 1:00:41 PM (2 occurrences)
Last logged: 2:25:03 PM

websocket connection is closing```

**Pull request merged**